source/AI.cpp

```diff
@@ -1789,7 +1798,13 @@ bool AI::MoveTo(Ship &ship, Command &command, const Point &targetPosition, const
 	if(!isClose || !isFacing)
 		command.SetTurn(TurnToward(ship, dp));
 	if(isFacing)
-		command |= Command::FORWARD;
+	{
```

Comment:
This section was updated in master - mind pulling in latest and verifying the behavior is as expected still?

Reply:
I did some more testing; when a ship has a velocity perpendicular to the target planet course, then the 30 degree margin results in the ship spiralling to its target planet. It doesn't look very bad, but it looks a bit unprofessional to see a ship spiral for landing.
I think this PR needs a little bit more work to fix this.

Reply:
I merged master so that anybody can run the test on the latest version.

Reply:
Spiralling is fixed. Ships now calculate the delta-vector they need to apply to get to their cruise-vector (cruise-speed in the direction of the target).